bitnami/clickhouse/templates/servicemonitor.yaml

```diff
@@ -12,6 +12,7 @@ metadata:
   {{- $labels := include "common.tplvalues.merge" ( dict "values" ( list .Values.metrics.serviceMonitor.labels .Values.commonLabels ) "context" . ) }}
   labels: {{- include "common.labels.standard" ( dict "customLabels" $labels "context" $ ) | nindent 4 }}
     app.kubernetes.io/component: clickhouse
+    app.kubernetes.io/part-of: clickhouse
```

Comment:
Does the keeper have a metrics endpoint?

Reply:
Yes, but it's exposed via the "normal" service

Reply:
Added a custom label

Reply:
done at https://github.com/bitnami/charts/pull/32958/commits/786dc3e86a09575f9fb564f1362f6c4b56c4e837